stdlib/buffer.gr

```diff
@@ -322,6 +322,18 @@ provide let addChar = (char, buffer) => {
   addString(Char.toString(char), buffer)
 }
 
+/**
+ * Appends a char from a code point to a buffer.
```

Comment:
```suggestion
 * Appends a character represented by a code point a buffer.
```

Reply:
I made this change, one note is should we refer to the `char` as a `character` in `addchar`? 
currently we say `Appends the bytes of a char to a buffer.`

Reply:
Sure, we use full words for descriptions, but that is unrelated to this PR.